build/source/engine/soilLiqFlx.f90

```diff
@@ -1263,112 +1157,76 @@ subroutine surfaceFlx(&
+     scalarInfilArea = min(0.5_rkind*(fInfRaw + sqrt(fInfRaw**2_i4b + scaleFactor)), 1._rkind)   ! infiltrating area -- constrained
     else
      scalarInfilArea = 1._rkind
-    endif
+    end if
 
     ! check to ensure we are not infiltrating into a fully saturated column
-    if(ixIce<nRoots)then
-     if(sum(mLayerVolFracLiq(ixIce+1:nRoots)*mLayerDepth(ixIce+1:nRoots)) > 0.9999_rkind*theta_sat*sum(mLayerDepth(ixIce+1:nRoots))) scalarInfilArea=0._rkind
-     !print*, 'ixIce, nRoots, scalarInfilArea = ', ixIce, nRoots, scalarInfilArea
-     !print*, 'sum(mLayerVolFracLiq(ixIce+1:nRoots)*mLayerDepth(ixIce+1:nRoots)) = ', sum(mLayerVolFracLiq(ixIce+1:nRoots)*mLayerDepth(ixIce+1:nRoots))
-     !print*, 'theta_sat*sum(mLayerDepth(ixIce+1:nRoots)) = ', theta_sat*sum(mLayerDepth(ixIce+1:nRoots))
-    endif
+    if (ixIce<nRoots) then
+     if (sum(mLayerVolFracLiq(ixIce+1:nRoots)*mLayerDepth(ixIce+1:nRoots)) > 0.9999_rkind*theta_sat*sum(mLayerDepth(ixIce+1:nRoots))) scalarInfilArea=0._rkind
+    end if
 
     ! define the impermeable area due to frozen ground
-    if(rootZoneIce > tiny(rootZoneIce))then  ! (avoid divide by zero)
+    if (rootZoneIce > tiny(rootZoneIce)) then  ! avoid divide by zero
      alpha            = 1._rkind/(soilIceCV**2._rkind)        ! shape parameter in the Gamma distribution
      xLimg            = alpha*soilIceScale/rootZoneIce  ! upper limit of the integral
-     !scalarFrozenArea = 1._rkind - gammp(alpha,xLimg)      ! fraction of frozen area
      scalarFrozenArea = 0._rkind
     else
      scalarFrozenArea = 0._rkind
     end if
-    !print*, 'scalarFrozenArea, rootZoneIce = ', scalarFrozenArea, rootZoneIce
+   end if ! end if desired to compute infiltration)
 
-   end if ! (if desire to compute infiltration)
-
-   ! compute infiltration (m s-1)
-   scalarSurfaceInfiltration = (1._rkind - scalarFrozenArea)*scalarInfilArea*min(scalarRainPlusMelt,xMaxInfilRate)
-
-   ! compute surface runoff (m s-1)
-   scalarSurfaceRunoff = scalarRainPlusMelt - scalarSurfaceInfiltration
-   !print*, 'scalarRainPlusMelt, xMaxInfilRate = ', scalarRainPlusMelt, xMaxInfilRate
-   !print*, 'scalarSurfaceInfiltration, scalarSurfaceRunoff = ', scalarSurfaceInfiltration, scalarSurfaceRunoff
-   !print*, '(1._rkind - scalarFrozenArea), (1._rkind - scalarFrozenArea)*scalarInfilArea = ', (1._rkind - scalarFrozenArea), (1._rkind - scalarFrozenArea)*scalarInfilArea
+   scalarSurfaceInfiltration = (1._rkind - scalarFrozenArea)*scalarInfilArea*min(scalarRainPlusMelt,xMaxInfilRate) ! compute infiltration (m s-1)
+   scalarSurfaceRunoff = scalarRainPlusMelt - scalarSurfaceInfiltration ! compute surface runoff (m s-1)
 
    ! set surface hydraulic conductivity and diffusivity to missing (not used for flux condition)
-   surfaceHydCond = realMissing
-   surfaceDiffuse = realMissing
+   surfaceHydCond = realMissing; surfaceDiffuse = realMissing
```

Comment:
Might be cleaner to keep these on separate lines

Reply:
I have reverted things to two separate lines for easy reading.